indico/modules/events/papers/blueprint.py

```diff
@@ -34,6 +34,12 @@
 
 # Management
 _bp.add_url_rule('/manage/papers/', 'management', management.RHPapersDashboard)
+_bp.add_url_rule('/manage/papers/deadlines/judging', 'manage_judging_deadline', management.RHSetJudgingDeadline,
```

Comment:
Why separate RHs? Could use `/deadlines/<any(judging,content,layout):type>` and then use `request.view_args['type']` in the RH